### ISSUE TITLE
Automate bigseo.save()

### ISSUE THREAD

**Issue by javieret** [grillorafael/bigseo#2]
I have been using the service for a couple of weeks and it works great, but now I'm facing a problem because. 

I'm posting a content from an app I made, so since it's in the app the content won't be cache until someone goes to the url of that content and "bigseo.save()" this command get executed while the page loads. 

Is there any way to programmatically execute bigseo.save?

**Comment by grillorafael** [comment]
Hi @javieret !

Do you want to save a specific page inside the POST method? That's what you wanna do? If so, there's no interface that you can use to do this but I can provide you one.

**Comment by javieret** [comment]
Yes, that's what I need. I actually saw the new commit, but still kind of confused on how to do it.

**Comment by grillorafael** [comment]
@javieret the last commit does not include this feature. I'll work on it today for you. I'll update you when it's done. :)
